doc/nrf/known_issues.rst

```diff
@@ -1391,25 +1404,55 @@ Reconnection issues after bonding
 Bluetooth mesh
 ==============
 
-.. rst-class:: v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
+.. rst-class:: v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-1 v1-9-0 v1-8-0 v1-7-1
 
-NCSDK-5580: nRF5340 only supports SoftDevice Controller
-  On nRF5340, only the :ref:`nrfxlib:softdevice_controller` is supported for Bluetooth® mesh.
+NCSDK-16800: RPL is not cleared on IV index recovery
+  After recovering the IV index, a node doesn't clear the replay protection list, which leads to incorrect triggering of the replay attack protection mechanism.
 
-.. rst-class:: v1-6-1 v1-6-0
+  **Workaround:** Call ``bt_mesh_rpl_reset`` twice after the IV index recovery is done.
 
-NCSDK-10200: The device stops sending Secure Network Beacons after re-provisioning
-  Bluetooth® mesh stops sending Secure Network Beacons if the device is re-provisioned after reset through Config Node Reset message or ``bt_mesh_reset()`` call.
+.. rst-class:: v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-1 v1-9-0 v1-8-0 v1-7-1
 
-  **Workaround:** Reboot the device after re-provisioning.
+NCSDK-16798: Friend Subsciption List may have duplicate entries
+  If a Low Power node loses a Friend Subscription List Add Confirm message, it repeats the request.
+  The Friend does not check both the transaction number and the presence of the addresses in the subscription list.
+  This causes a situation where the Friend fills the subscription list with duplicate addresses.
+
+.. rst-class:: v2-1-0 v2-0-2 v2-0-1 v2-0-0
+
+NCSDK-16782: The extended advertiser may not work with Bluetooth mesh
+  Using the extended advertiser instead of the legacy advertiser can lead to getting composition data while provisioning to fail.
```

Comment:
Add this text: "This problem may manifest in the sample :ref:`bluetooth_ble_peripheral_lbs_coex`, as it is using the extended advertiser."

Reply:
Added.